analysis_options.yaml

```
@@ -4,15 +4,16 @@ analyzer:
     implicit-casts: false
 linter:
   rules:
+    - avoid_dynamic_calls
     - avoid_empty_else
     - avoid_init_to_null
     - avoid_null_checks_in_equality_operators
     - avoid_unused_constructor_parameters
     - await_only_futures
     - camel_case_types
     - cancel_subscriptions
-    #- constant_identifier_names
     - control_flow_in_finally
+    - constant_identifier_names
     - directives_ordering
```

Might as well add this one in also :)

```suggestion
    - depend_on_referenced_packages
    - directives_ordering
```